CIP-1852/CIP-1852.md

```diff
@@ -50,11 +50,12 @@ Example: `m / 1852' / 1815' / 0' / 0 / 0`
 
 Here, `chain` can be the following
 
-| Name           | Value | Description
-|----------------|-------|-------------
-| External chain | `0`   | Same as defined in [BIP44](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki)
-| Internal chain | `1`   | Same as defined in [BIP44](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki)
-| Staking Key    | `2`   | See [CIP11](../CIP-0011)
+| Name               | Value | Description
+|--------------------|-------|-------------
+| External chain     | `0`   | Same as defined in [BIP44](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki)
+| Internal chain     | `1`   | Same as defined in [BIP44](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki)
+| Staking Key        | `2`   | See [CIP11](../CIP-0011)
+| Collateral Account | `3`   | See [CIP11](../CIP-???)
 
 Wallets **MUST** implement this new scheme using the master node derivation algorithm from Icarus with sequential addressing (see [CIP3](../CIP-0003) for more information)
```

Comment:
this requirement isn't currently feasible for Ledger as the Cardano app doesn't have control over the master node derivation (and that's why it wasn't compliant with Icarus so far)

Reply:
I'm not sure this way of treating collateral is relevant any longer. Babbage will come with collateral outputs
https://hydra.iohk.io/build/11759383/download/1/babbage-changes.pdf
and then collateral does not need to be treated separately, any utxos (without datum hashes) would do as collateral inputs, even those including tokens.